aspnetcore/fundamentals/servers/kestrel/sample2/Startup.cs

```diff
@@ -0,0 +1,94 @@
+﻿#define Default // or Limits
+
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Hosting.Server.Features;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Http.Extensions;
+using Microsoft.AspNetCore.Http.Features;
+using Microsoft.AspNetCore.Server.Kestrel.Core;
+using Microsoft.AspNetCore.Server.Kestrel.Core.Features;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+using System;
+
+namespace KestrelDemo
+{
+    public class Startup
+    {
+        public Startup(IHostingEnvironment env)
+        {
+            Configuration = new ConfigurationBuilder()
+                .AddEnvironmentVariables()
+                .Build();
+        }
+
+        public IConfigurationRoot Configuration { get; private set; }
```

Comment:
Is this `Configuration` property or the Startup constructor necessary for this sample? It looks like it can be removed.

At the very least, the `IHostingEnvironment` parameter can be removed from the constructor.